source-code/plugins/i18next/examples/example02/inlang.config.json

```diff
@@ -1,6 +1,6 @@
 {
 	"sourceLanguageTag": "en",
-	"languageTags": ["en"],
+	"languageTags": ["en", "de"],
 	"modules": ["./dist/index.js"],
 	"settings": {
 		"plugins": {
```

Comment:
@inlang/plugins can we get rid of the "exampleXX" pattern? Testing what needs to be tested in unit tests should be possible. 

- nobody known what is tested if the example is called `example01`
- using regular unit test patterns make the test easier to follow